package.json

```diff
@@ -15,7 +15,7 @@
 	},
 	"scripts": {
 		"install": "lerna bootstrap --force-local --hoist",
-		"start": "export NODE_OPTIONS='--max-old-space-size=4096' && react-app-rewired start",
+		"start": "export NODE_OPTIONS='--max-old-space-size=4096' && npm run clean && concurrently \"react-app-rewired start\" \"chokidar './src/**/readme.tsx' -c 'npm run docgen:examples'\"",
```

Comment:
- Cleaning the directory before npm start so we don't run into the issue of our dev experiencing failing from built files
- Including concurrently and chokidar to watch readme files and recompile them. This means we get live updates of the docsite when running npm start. Note, this is only the examples, the prop tables won't update in development because it's an expensive operation to run.